### ISSUE TITLE
100% CPU

### ISSUE THREAD

**Issue by Gapstare** [rroart/freevms#2]
Upon launching FreeVMS with qemu-system-i386, the qemu process spikes to 100% CPU usage. Is this normal? :-)

**Comment by rroart** [comment]
Yes, this is currently normal, but it is likely something that will have to be fixed.
Thanks for reminding.

Sorry for a very late answer, but I have been very busy with other projects/things, and not  this one anymore.

**Comment by Gapstare** [comment]
Ok, I'll keep calm about it for now :) Thanks for your answer.
